swift/src/Ice/CommunicatorI.swift

```diff
@@ -14,12 +14,14 @@ class CommunicatorI: LocalObject<ICECommunicator>, Communicator, @unchecked Send
         self.initData = initData
         do {
             defaultsAndOverrides = try DefaultsAndOverrides(initData.properties!)
-            classGraphDepthMax = try initData.properties!.getIcePropertyAsInt("Ice.ClassGraphDepthMax")
+            classGraphDepthMax = try initData.properties!.getIcePropertyAsInt(
```

Comment:
You should undo some changes in this file. Looks like a config issue with your Swift formatter.